backend/apps/uploads/views.py

```
@@ -225,18 +225,57 @@ def post(self, request):
+        except ValidationError as exc:
+            return Response({"error": exc.messages[0]}, status=status.HTTP_400_BAD_REQUEST)
+
         valid_name = get_valid_filename(uploaded_file.name)
-        media_root = Path(getattr(settings, "MEDIA_ROOT", settings.BASE_DIR / "media"))
-        upload_dir = media_root / "uploads"
-        upload_dir.mkdir(parents=True, exist_ok=True)
+        quarantine_root = Path(
+            getattr(settings, "UPLOAD_QUARANTINE_ROOT", settings.BASE_DIR / "quarantine")
+        )
+        quarantine_root.mkdir(parents=True, exist_ok=True)
+        quarantine_path = quarantine_root / f"{uuid.uuid4()}_{valid_name}"
```

Ruff (0.16.0)

[error] Undefined name `uuid`

(F821)